content/en/docs/languages/php/instrumentation.md

```diff
@@ -150,104 +290,308 @@ function as part of PHP's shutdown process:
-[`Tracer`](/docs/concepts/signals/traces/#tracer).
+If a `TracerProvider` is not created, the OpenTelemetry APIs for tracing will
+use a no-op implementation and fail to generate data. As explained next, modify
+the `instrumentation.ts` (or `instrumentation.js`) file to include all the SDK
+initialization code in Node and the browser.
 
-A `Tracer` is responsible for creating spans and interacting with the
-[Context](../propagation/). A tracer is acquired from a `TracerProvider`,
-specifying the name and other (optional) identifying information about the
-[library instrumenting](/docs/concepts/instrumentation/libraries/) the
-instrumented library or application to be monitored.
+If you followed the instructions to [initialize the SDK](#initialize-the-sdk)
+above, you have a `TracerProvider` setup for you already. You can continue with
+[acquiring a tracer](#acquiring-a-tracer).
+
+### Acquiring a Tracer
 
-More information is available in the specification chapter
-[Obtaining a Tracer](/docs/specs/otel/trace/api/#tracerprovider).
+Anywhere in your application where you write manual tracing code should call
+`getTracer` to acquire a tracer. For example:
 
 ```php
 $tracerProvider = Globals::tracerProvider();
 $tracer = $tracerProvider->getTracer(
-  'instrumentation-library-name', //name (required)
-  '1.0.0', //version
+  'instrumentation-scope-name', //name (required)
+  'instrumentation-scope-version', //version
   'http://example.com/my-schema', //schema url
   ['foo' => 'bar'] //attributes
 );
 ```
 
-Important: the parameters used when acquiring a tracer are purely
-informational - these values will be emitted as part of the scope of any
-telemetry emitted by that tracer. All `Tracer`s that are created by a single
-`OpenTelemetry` instance will interoperate, regardless of differences in these
-parameters.
+The values of `instrumentation-scope-name` and `instrumentation-scope-version`
+should uniquely identify the
+[Instrumentation Scope](/docs/concepts/instrumentation-scope/), such as the
+package, module or class name. While the name is required, the version is still
+recommended despite being optional.
+
+It's generally recommended to call `getTracer` in your app when you need it
+rather than exporting the `tracer` instance to the rest of your app. This helps
+avoid trickier application load issues when other required dependencies are
+involved.
+
+In the case of the [example app](#example-app), there are two places where a
+tracer may be acquired with an appropriate Instrumentation Scope:
+
+First, in the _application file_ `index.php`:
+
+```php
+<?php
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Log\LogLevel;
+use Slim\Factory\AppFactory;
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use OpenTelemetry\API\Globals;
+
+require __DIR__ . '/vendor/autoload.php';
+
+require('dice.php');
+require('instrumentation.php');
```

Comment:
this might be a js/node copy-paste. I don't see any indication of what might be in this file, but presumably the setup code to create and register providers?